modules/core/04-channel/keeper/packet.go

```diff
@@ -461,6 +469,10 @@ func (k Keeper) AcknowledgePacket(
 
 	commitment := k.GetPacketCommitment(ctx, packet.GetSourcePort(), packet.GetSourceChannel(), packet.GetSequence())
 
+	if len(commitment) == 0 {
+		return sdkerrors.Wrapf(types.ErrPacketAcknowledged, "packet commitment does not exist")
+	}
```

Comment:
Are we sure this means packet was acknowledged? What if it never existed in the first place?